api/libretime_api/storage/models/file.py

```diff
@@ -2,82 +2,122 @@
     length = models.DurationField(blank=True, null=True)
+
+    # Metadata
+    bpm = models.IntegerField(blank=True, null=True)
+    replay_gain = models.DecimalField(
+        max_digits=8,
+        decimal_places=2,
+        blank=True,
+        null=True,
+    )
+
+    name = models.CharField(max_length=255)  # Unused ?
+    description = models.CharField(
+        max_length=512,
+        blank=True,
+        null=True,
+    )  # Replace comment ?
+
+    artist_name = models.CharField(max_length=512, blank=True, null=True)
     album_title = models.CharField(max_length=512, blank=True, null=True)
+    track_title = models.CharField(max_length=512, blank=True, null=True)
     genre = models.CharField(max_length=64, blank=True, null=True)
-    comments = models.TextField(blank=True, null=True)
     year = models.CharField(max_length=16, blank=True, null=True)
     track_number = models.IntegerField(blank=True, null=True)
-    channels = models.IntegerField(blank=True, null=True)
-    url = models.CharField(max_length=1024, blank=True, null=True)
-    bpm = models.IntegerField(blank=True, null=True)
-    rating = models.CharField(max_length=8, blank=True, null=True)
-    encoded_by = models.CharField(max_length=255, blank=True, null=True)
     disc_number = models.CharField(max_length=8, blank=True, null=True)
-    mood = models.CharField(max_length=64, blank=True, null=True)
-    label = models.CharField(max_length=512, blank=True, null=True)
+    # https://github.com/libretime/libretime/issues/526
+    comment = models.TextField(db_column="comments", blank=True, null=True)
     composer = models.CharField(max_length=512, blank=True, null=True)
+    conductor = models.CharField(max_length=512, blank=True, null=True)
+    copyright = models.CharField(max_length=512, blank=True, null=True)
+    language = models.CharField(max_length=512, blank=True, null=True)
     encoder = models.CharField(max_length=64, blank=True, null=True)
-    checksum = models.CharField(max_length=256, blank=True, null=True)
-    lyrics = models.TextField(blank=True, null=True)
+    encoded_by = models.CharField(max_length=255, blank=True, null=True)
+    isrc_number = models.CharField(max_length=512, blank=True, null=True)
+    label = models.CharField(max_length=512, blank=True, null=True)
+    url = models.CharField(max_length=1024, blank=True, null=True)
+    mood = models.CharField(max_length=64, blank=True, null=True)
     orchestra = models.CharField(max_length=512, blank=True, null=True)
-    conductor = models.CharField(max_length=512, blank=True, null=True)
+    lyrics = models.TextField(blank=True, null=True)
     lyricist = models.CharField(max_length=512, blank=True, null=True)
     original_lyricist = models.CharField(max_length=512, blank=True, null=True)
-    radio_station_name = models.CharField(max_length=512, blank=True, null=True)
+
+    rating = models.CharField(max_length=8, blank=True, null=True)
```

Comment:
Again, unsure where this is used and how it is filled. It was empty in the databases I queried.

Reply:
Probably more general metadata fields that mutagen provides if it is set in the ID3 tags?

Reply:
Yup, we will have to let the analyzer fill this field in that case.